analyzere/utils.py

```diff
@@ -92,3 +92,17 @@ def vectorize(values):
     if isinstance(values, list):
         return ','.join(str(v) for v in values)
     return values
+
+
+def vectorize_range(values):
```

Comment:
I'm not sure I like `vectorize_range()` as a the name. Maybe `vectorize_windows()` or `vectorize_probability_windows()`?